milli/src/search/mod.rs

```diff
@@ -152,14 +152,15 @@ impl<'a> Search<'a> {
         mut distinct: D,
         matching_words: MatchingWords,
         mut criteria: Final,
-    ) -> anyhow::Result<SearchResult>
-    {
+    ) -> anyhow::Result<SearchResult> {
         let mut offset = self.offset;
         let mut initial_candidates = RoaringBitmap::new();
         let mut excluded_candidates = RoaringBitmap::new();
         let mut documents_ids = Vec::with_capacity(self.limit);
 
-        while let Some(FinalResult { candidates, bucket_candidates, .. }) = criteria.next(&excluded_candidates)? {
+        while let Some(FinalResult { candidates, bucket_candidates, .. }) =
+            criteria.next(&excluded_candidates)?
+        {
```

Comment:
@irevoire here is an example of why I would like to keep this parameter, if it wasn't specified it would have resulted in something like that:

```
while let Some(FinalResult {
        candidates,
        bucket_candidates,
        ..
    }) = criteria.next(&excluded_candidates)? {
```